core/src/utils/test/playwright/page/utils/set-content.ts

```diff
@@ -42,7 +42,7 @@ export const setContent = async (page: Page, html: string, testInfo: TestInfo, o
         <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
         <link href="${baseUrl}/css/ionic.bundle.css" rel="stylesheet" />
         <link href="${baseUrl}/scripts/testing/styles.css" rel="stylesheet" />
-        ${theme !== 'light' ? `<link href="${baseUrl}/src/themes/test/${theme}.css" rel="stylesheet" />` : ''}
+        ${theme !== 'light' ? `<link href="${baseUrl}/css/themes/${theme}.always.css" rel="stylesheet" />` : ''}
```

Comment:
Our standard contrast dark theme can benefit from this change, so I put up https://github.com/ionic-team/ionic-framework/pull/28795 to change this in `feature-8.0`. Once that is merged in I'll sync this branch with `feature-8.0`.